cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/TestCommandTest.java

```diff
@@ -277,8 +278,9 @@ public void testBalTestWithStickyFlag() throws IOException {
         String buildLog = readOutput(true);
         Assert.assertEquals(buildLog.replaceAll("\r", ""), getOutput("bal-test-project.txt"));
         Assert.assertTrue(projectPath.resolve(DEPENDENCIES_TOML).toFile().exists());
-        Assert.assertEquals(readFileAsString(projectPath.resolve(DEPENDENCIES_TOML)).trim(), readFileAsString(
-                projectPath.resolve(RESOURCE_DIR_NAME).resolve("expectedDeps.toml")).trim());
+        Assert.assertEquals(readFileAsString(projectPath.resolve(DEPENDENCIES_TOML)).trim(),
+                readFileAsString(projectPath.resolve(RESOURCE_DIR_NAME).resolve("expectedDeps.toml"))
```

Comment:
can we use a constant for `expectedDeps.toml`  as we are using it in multiple places?